server/capabilities/gke_networks_endpoint.go

```diff
@@ -0,0 +1,89 @@
+package capabilities
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"google.golang.org/api/compute/v1"
```

Comment:
Put this import below the golang imports separated by a space, like:
```
        "context"
 	"encoding/json"
 	"fmt"
        "net/http"

 	"google.golang.org/api/compute/v1"
```

Reply:
Updated!